platform/linux-generic/odp_pkt_queue.c

```diff
@@ -104,6 +114,13 @@ static int pkt_queue_free_list_add(queue_pool_t *pool,
 		if (!queue_blks) {
 			malloc_len = num_blks * sizeof(queue_blk_t);
 			queue_blks = malloc(malloc_len);
```

Comment:
No i did not see any allocation failures in run-time, i just noticed the malloc without checks.